src/chat/itemCard.ts

```diff
@@ -120,7 +122,7 @@ export class ItemCard {
       const speakerName = getSpeakerName(flags.actorId, flags.tokenId);
       await ItemCard.roll(flags.itemData, modifier, actor, speakerName);
     } else {
-      ui.notifications?.warn("Could not find world actor to do ability roll");
+      ui.notifications?.warn(localize("RQG.Item.Notifications.CouldNotFindActorByIdsWarn", {actorId: flags.actorId, tokenId: flags.tokenId}));
```

Comment:
```suggestion
      ui.notifications?.warn(localize("RQG.Item.Notification.CouldNotFindActorByIdsWarn", {actorId: flags.actorId, tokenId: flags.tokenId}));
```

Reply:
Again, looks like I did this already..  I think I had the same thought before I even saw this.